### PR TITLE
histogram aggregation test improved.

### PR DESCRIPTION
**Pull request by hkulekci**
- Histogram aggregation tests improved. 
 - Aggregation test broken namespaces fixed. 
 - HistogramAggregation extended_bounds deprecation error fixed for `strlen` function

### CODE REVIEW

**Comment by hkulekci** [comment]
@alexander-schranz Ping

**Comment by hkulekci** [comment]
Hello @alexander-schranz, could you check this PR?

File: src/Aggregation/Bucketing/HistogramAggregation.php [hunk 1]
```diff
@@ -203,7 +203,9 @@ public function setExtendedBounds($min = null, $max = null)
                 'min' => $min,
                 'max' => $max,
             ],
-            'strlen'
+            static function ($item) {
+                return $item !== null;
+            }
```
Comment (alexander-schranz) [inline src/Aggregation/Bucketing/HistogramAggregation.php:208]: Looks like this should also be fixed for 7.x version. Or does it only appear on 8.x?

**Review by alexander-schranz — CHANGES_REQUESTED** [review]
Sorry for the late response.

File: src/Aggregation/Bucketing/HistogramAggregation.php (hunk 1 above)
Comment (hkulekci) [reply src/Aggregation/Bucketing/HistogramAggregation.php:208]: I could not remember exactly, but I think it should be.

Comment (alexander-schranz) [reply src/Aggregation/Bucketing/HistogramAggregation.php:208]: Can you then rebase the PR to 7.x then?

**Comment by alexander-schranz** [comment]
fixed in #24 and merged back to 8.x.

@hkulekci Thank you!

**Comment by alexander-schranz** [comment]
Tagged as 8.0.1

**Pull request closed without merging**